Docker/Image/Builder/ImageBuilder.php

```diff
@@ -271,7 +271,7 @@ private function handleGitCredentials($path)
     private function createDockerFile()
     {
         $dockerFile = '';
-        $dockerFile .= "FROM " . $this->getImageId() . "\n";
+        $dockerFile .= "FROM " . $this->getImageId() . ":" . $this->getTag() . "\n";
```

Comment:
jo, tak to asi nejak vzniklo - ze ten getFullImageId v builderu dela neco trochu jinyho nez jinde, takhle je to asi ok